.github/workflows/release.yml

```diff
@@ -20,15 +20,14 @@ jobs:
         with:
           ruby-version: 3.2
         if: ${{ steps.release.outputs.release_created }}
-      - name: Publish gem to Github repository packages
+      - name: Publish gem to Rubygems
         run: |
           mkdir -p $HOME/.gem
           touch $HOME/.gem/credentials
           chmod 0600 $HOME/.gem/credentials
-          printf -- "---\n:github: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
+          printf -- "---\n:rubygems: ${GEM_HOST_API_KEY}\n" > $HOME/.gem/credentials
           gem build *.gemspec
-          gem push --KEY github --host https://rubygems.pkg.github.com/${OWNER} *.gem
+          gem push --KEY rubygems --host https://rubygems.org *.gem
```

Comment:
This could be simplified given rubygems.org is the default repo and having the secret in the environment should be enough, but I'm keeping it as similar as possible to github package publishing setup for coherence.